src/Symfony/Bridge/Twig/Command/LintCommand.php

```diff
@@ -17,6 +17,11 @@
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Finder\Finder;
+use Twig\Environment;
+use Twig\Error\Error;
+use Twig\Loader\ArrayLoader;
+use Twig\Source;
```

Comment:
@keradus just wondering: this `use`is not used in the class - is it a php-cs-fixer bug that fabbot is still green?